modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -249,6 +250,12 @@ public TransactionConfig getTransactionConfig() {
       public ClassLoader getExecutionClassLoader() {
         return muleContext.getExecutionClassLoader();
       }
+
+      @Override
+      public ComponentIdentifier getSourceIdentifier() {
```

Comment:
are these ComponentIdentifiers something we can use to replaced the current coordinates system in the DataSense API?